content/posts/newsletter-015/index.md

```diff
@@ -60,6 +60,20 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Camp Misty]
```

Comment:
These square brackets don't have a corresponding reference-style link, so they're showing up as literal square brackets in the post - if you look at the [example](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/source/CONTRIBUTING.md#style), there's a corresponding URL at the bottom of the section to go with the title.

Can you either fix this link or remove the square brackets?

Reply:
Woops! I'm at work right now, but I'll fix it when I get home.